### DISCUSSION TITLE
Change Image Size on Recipes Page/ Duplicate Recipes

### DISCUSSION THREAD

**Discussion by CMoJO83** [TandoorRecipes/recipes#3104] [Q&A]
Hi! First off, I'm not a coder so sorry if this is a dumb set of questions... my husband is and he got me introduced to Tandoor because Pinterest has become a dumpster fire of an app which is no longer useful for me to manage recipes. Loving Tandoor and all its features so far but I had two questions:

1) Is there a way to increase the height of the images on the main recipes page? I'm sure there is some code we can change on our end to adjust the pixel height, but I don't know how to really get there to change it up. I'd just like to see more of the image so I can visually parse things more easily when scrolling through my items. I'm pretty sure with a little guidance I can figure it out if that's even an option.

2) Is there a way to have the app notify you if you try to import a recipe from the web that you've already imported? I'm working through transferring things pretty methodically, but I know at some point I'm going to add the same recipe twice. Just curious if there is a way to prevent that.

Thanks!! 
- CJ

**Comment by vabene1111** [comment]
hi, thanks for your feedback.

1. not really, I am currently working on a new frontend that will improve how the image is displaye. You could play around with custom CSS but I recommend against that as it would be pretty hacky.
2. I ran into this issue a few times as well. Currently there isn't but its actually somewhat easy to implement because I save the imports url, so maybe during the new frontend rewrite I will add an option to prevent duplicates.

**Reply by CMoJO83** [reply]
Okay, thank you for the reply. I'll keep an eye out for new versions as you release them. I absolutely love what you've done so far with the app; your work is definitely appreciated!

**Comment by jeko89** [comment]
Otherwise could you add a photo resize wizard in the main recipe image upload and permit to open the whole image with a mouse click on it? Moreover I must always press SAVE 2 times after uploading a main recipe photo (it means AT LEAST 30-40s of processing after each SAVE press that doesn't depend on photo bit size). By the way, a part for image support, it's a great app, thank you @vabene1111!

**Reply by vabene1111** [reply]
yes there is currently a bug with the save there, I need to fix that. It should not take this long to save tough, are you using sqlite or postgres?
